drivers/SmartThings/matter-sensor/profiles/air-quality-sensor-temp-humidity-all-level-all-meas.yml

```diff
@@ -0,0 +1,54 @@
+name: air-quality-sensor-temp-humidity-all-level-all-meas
+components:
+  - id: main
+    capabilities:
+      - id: airQualityHealthConcern
+        version: 1
+      - id: temperatureMeasurement
+        version: 1
+      - id: relativeHumidityMeasurement
+        version: 1
+      - id: carbonMonoxideMeasurement
+        version: 1
+      - id: carbonMonoxideHealthConcern
+        version: 1
+      - id: carbonDioxideMeasurement
+        version: 1
+      - id: carbonDioxideHealthConcern
+        version: 1
+      - id: nitrogenDioxideMeasurement
+        version: 1
+      - id: nitrogenDioxideHealthConcern
+        version: 1
+      - id: ozoneMeasurement
+        version: 1
+      - id: ozoneHealthConcern
+        version: 1
+      - id: formaldehydeMeasurement
+        version: 1
+      - id: formaldehydeHealthConcern
+        version: 1
+      - id: veryFineDustSensor
+        version: 1
+      - id: veryFineDustHealthConcern
+        version: 1
+      - id: fineDustSensor
+        version: 1
+      - id: fineDustHealthConcern
+        version: 1
+      - id: dustSensor
+        version: 1
+      - id: dustHealthConcern
+        version: 1
+      - id: radonMeasurement
+        version: 1
+      - id: radonHealthConcern
+        version: 1
+      - id: tvocMeasurement
+        version: 1
+      - id: tvocHealthConcern
+        version: 1
+      - id: refresh
+        version: 1
+    categories:
+    - name: AirQualityDetector
```

Comment:
I dont believe we have profiles to date `category: AirQualityDetector`. Could you confirm that the device Icon